test/e2e/models/CodeServer.ts

```diff
@@ -13,14 +13,21 @@ interface CodeServerProcess {
   address: string
 }
 
-class CancelToken {
+class Context {
```

Comment:
❓ What was the reason for changing this?

Reply:
I am using it to signal both canceling and being done so I felt `CancelToken` no longer fit.